docs/build/advanced-concepts/metadata-views.md

```diff
@@ -97,50 +246,146 @@ case Type<MetadataViews.Display>():
+The `Serial` metadata provides the unique serial number of the NFT,
+akin to a serial number on a currency note or a VIN on a car.
+This serial number is a fundamental attribute that certifies the individuality
+of each NFT and is critical for identification and verification processes.
+Serial numbers are expected to be unique among other NFTs from the same project.
+Many projects are already using the NFT resource's globally unique UUID as the ID already,
+so they will typically also use that as the serial number.
 
 ```cadence
 case Type<MetadataViews.Serial>():
-    return MetadataViews.Serial(self.id)
+    return MetadataViews.Serial(self.uuid)
 ```
 
 ### Royalties Metadata
 
-Royalty information is vital for the sustainable economics of the creators in the NFT space. The `Royalties` metadata view defines the specifics of any royalty agreements in place, including the percentage of sales revenue that will go to the original creator or other stakeholders on secondary sales.
+Royalty information is vital for the sustainable economics of the creators in the NFT space.
+[The `Royalties` metadata view](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L295)
+defines the specifics of any royalty agreements in place,
+including the percentage of sales revenue that will go to the original creator
+or other stakeholders on secondary sales.
 
+Each royalty view contains a fungible token receiver capability where royalties should be paid:
+```cadence
+pub struct Royalty {
+
+    pub let receiver: Capability<&AnyResource{FungibleToken.Receiver}>
+
+    pub let cut: UFix64
+}
+```
+
+here is an example of how an NFT might return a `Royalties` view:
 ```cadence
 case Type<MetadataViews.Royalties>():
     // Assuming each 'Royalty' in the 'royalties' array has 'cut' and 'description' fields
-    let detailedRoyalties = self.royalties.map { royalty ->
+    let royalty =
         MetadataViews.Royalty(
-            receiver: royalty.receiver,      // The beneficiary of the royalty
-            cut: royalty.cut,                // The percentage cut of each sale
-            description: royalty.description // A description of the royalty terms
+            // The beneficiary of the royalty: in this case, the contract account
+            receiver: ExampleNFT.account.getCapability<&AnyResource{FungibleToken.Receiver}>(/public/GenericFTReceiver),
+            // The percentage cut of each sale
+            cut: 0.05,
+            // A description of the royalty terms          
+            description: "Royalty payment to the original creator"
         )
     }
     return MetadataViews.Royalties(detailedRoyalties)
 ```
 
+If someone wants to make a listing for their NFT on a marketplace,
+the marketplace can check to see if the royalty receiver
+accepts the seller's desired fungible token by calling the `isSupportedVaultType()`
```

Comment:
I believe this is a v2 `Receiver` interface method. The [currently deployed FT contract](https://contractbrowser.com/A.f233dcee88fe0abe.FungibleToken) defines `receiver.getSupportedVaultTypes(): {Type: Bool}`.

Reply:
okay. I'll update that